pkg/deploy/config.go

```diff
@@ -41,6 +41,7 @@ type Configuration struct {
 	ARMAPICABundle                     *string       `json:"armApiCaBundle,omitempty"`
 	ARMAPIClientCertCommonName         *string       `json:"armApiClientCertCommonName,omitempty"`
 	ARMClientID                        *string       `json:"armClientId,omitempty"`
+	AzSecPackVSATenantId               *string       `json:"azureSecPackVSATenantId,omitempty"`
```

Comment:
Your test is failing because the parameter you have is `azSecPackVSATenantID` instead of what's in the config here as `azureSecPackVSATenantId`.  Even though it's in the config, it can not resolve the parameter set on the config.

Reply:
For consistency, it may make sense to use `azureSecPackVSATenantId` in the `deploy/rp-production-parameters.json`

Reply:
changing to azure, updated rp-config too https://msazure.visualstudio.com/AzureRedHatOpenShift/_git/RP-Config/commit/bcf3556c3a9793e25cd3c937791873a6b5d01c98?refName=refs%2Fheads%2Fazsecpack_offnode